src/index.ts

```diff
@@ -18,16 +27,44 @@ const formatError = winston.format(info => {
     return info;
 });
 
+/* A custom format that is used to include tenantId. */
+const formatTenantId = (parameters: ParameterConfig | undefined) => {
+    return winston.format(info => {
```

Comment:
Can we remove the dependency on the tenantId and make this generic?
From the parameters config, get the keys, check if the key is present in the store, if yes, add it to `info` else skip.